### ISSUE TITLE
Scanner Results Leading 0's

### ISSUE THREAD

**Issue by KittoniuM** [KN4CK3R/ReClass.NET#35]
https://i.imgur.com/oPUPVzY.png

would look better if the 0's got removed

**Comment by Timboy67678** [comment]
i disagree with this, i think the zero's let you know quickly at what level the adresss is at.

Also, no offence, but instead of making so many issues for feature requests, how about trying to add them yourself? something like this is trivial at best

**Comment by KittoniuM** [comment]
never rly messed with C# other than one time in 2014 and would be better if it was official

**Comment by KN4CK3R** [comment]
I agree with @Timboy67678 but I will add a settings tab for the memory scanner to make it flexible.
